src/main/java/com/dani/blog/domain/post/PostTransformer.java

```diff
@@ -0,0 +1,21 @@
+package com.dani.blog.domain.post;
+
+import com.dani.blog.base.service.ModelMapperTransformer;
+import com.dani.blog.domain.Post;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+/**
+ * @author dani
+ */
+@Component
+public class PostTransformer extends ModelMapperTransformer<PostEntity, Post> {
+
+    @Inject
+    public PostTransformer(ModelMapper mapper) {
```

Comment:
Would that be still valid? I'm a bit confused, since i'm annotation `PostTransformer` only with `@Component`, how `mapper` would be injected into `ModelMapperTransformer` if i'm calling it's constructor from PostTransformer's?